airbyte-webapp-e2e-tests/cypress/commands/connection.ts

```diff
@@ -13,13 +13,23 @@ export const createTestConnection = (sourceName: string, destinationName: string
       createPokeApiSource(sourceName, "luxray")
       break;
     case sourceName.includes('Postgres'):
-      createPostgresSource(sourceName);
+      createPostgresSource(sourceName, "ec2-3-229-11-55.compute-1.amazonaws.com", "{selectAll}{del}5432", "d5pcodgtmh9pob", "yuxothkqynerju", "1c1771f795985693bc4f780130d7e8af895a3d48155007b93d21218e4bc07cd9", "demo");
```

Comment:
Is this a PAT? CC @timroes

Reply:
We have integration test data that we somehow inject into integration tests for connectors.

There are two options here, we can either use a hosted Postgres here as well and inject that data in a similar matter, which has the disadvantage that this won't work for OSS users anymore, since only we can run those tests in our infrastructure. Alternatively, we could just export a SQL dump from the data we want and make sure we inject that before running our tests into our locally running Postgres so we always have the same data, and not use hosted test instances unless really needed (because we might want to test an API service).

@ambirdsall I'd refer to you here once you're back for more clarification and setting directions.

Reply:
Unless there's some context I'm missing, I have a pretty solid preference for the second solution (using a SQL dump to insert known test data in a local postgres database) to keep this working as well as possible for all external open-source contributors. It would be fairly straightforward to define a `"postcreatedb"` npm script to load the SQL dump; this would be automatically run after `npm run createdb`. I haven't had a chance to test this yet, but I would expect changing `airbyte-webapp-e2e-tests/package.json` with something like the following would work:
```diff
-    "createdb": "docker run --rm -d -p 5433:5432 -e POSTGRES_PASSWORD=secret_password -e POSTGRES_DB=airbyte_ci --name airbyte_ci_pg postgres"
+    "createdb": "docker run --rm -d -p 5433:5432 -e POSTGRES_PASSWORD=secret_password -e POSTGRES_DB=airbyte_ci --name airbyte_ci_pg postgres",
+    "postcreatedb": "docker run --rm -d -p 5433:5432 -e POSTGRES_PASSWORD=secret_password -e POSTGRES_DB=airbyte_ci --name airbyte_ci_pg psql --single-transaction < test-data.sql"
},
```

Do we have any documentation or reference for the desired test data? If not, dumping the remote database that is currently configured in this PR seems like a reasonable starting point.